graphql/version_resolver.go

```diff
@@ -518,6 +518,39 @@ func (r *versionResolver) Warnings(ctx context.Context, obj *restModel.APIVersio
 	return v.Warnings, nil
 }
 
+// WaterfallBuilds is the resolver for the waterfallBuilds field.
+func (r *versionResolver) WaterfallBuilds(ctx context.Context, obj *restModel.APIVersion) ([]*model.WaterfallBuild, error) {
+	versionId := utility.FromStringPtr(obj.Id)
+	// If activated is nil in the db we should resolve it and cache it for subsequent queries. There is a very low likely hood of this field being hit
+	if obj.Activated == nil {
+		version, err := model.VersionFindOne(model.VersionById(versionId))
+		if err != nil {
+			return nil, InternalServerError.Send(ctx, fmt.Sprintf("fetching version '%s': %s", versionId, err.Error()))
+		}
+		if err = setVersionActivationStatus(ctx, version); err != nil {
+			return nil, InternalServerError.Send(ctx, fmt.Sprintf("setting version activation status: %s", err.Error()))
+		}
+		obj.Activated = version.Activated
+	}
```

Comment:
Is this still necessary? We introduced the activated field a [few years ago](https://github.com/evergreen-ci/evergreen/pull/4626) and have since introduced the TTL I don't think any versions that haven't been TTL'd should still have this field as nil.

Reply:
Thanks for the context, this was copy pasta. Removed!